pkg/api/pluginproxy/pluginproxy_test.go

```diff
@@ -142,7 +169,7 @@ func getPluginProxiedRequest(ctx *models.ReqContext, cfg *setting.Cfg, route *pl
 	}
 	proxy := NewApiPluginProxy(ctx, "", route, "", cfg)
 
-	req, err := http.NewRequest(http.MethodGet, route.URL, nil)
+	req, err := http.NewRequest(http.MethodGet, "", nil)
```

Comment:
Did this cause a problem? Just trying to understand why you changed it and/or if it really doesn't matter.

Reply:
Yes, the issue I was running into was that when `route.URL` is an interpolated string, the call to `http.NewRequest` would fail to parse the URL parameter with the following error (`parse "{{if .JsonData.apiHost}}{{.JsonData.apiHost}}{{else}}https://example.com{{end}}": first path segment in URL cannot contain colon`).

This is somewhat confusing. The `req` object is the request that is being proxied, so in practice the URL would actually be along the lines of `/api/plugin-proxy/grafana-simple-app/api/v4/endpoint`. I went ahead and changed from an empty string to something like that URL to hopefully make that clearer.  Let me know if you think a comment would be better (or any other suggestions you might have).